tests/integration/targets/vmware_datacenter_info/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+# Test code for the vmware_datacenter_info module.
+# Copyright: (c) 2020, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+
+- name: Gather information about all datacenter
+  vmware_datacenter_info: &vmware_dc_data
+    validate_certs: False
```

Comment:
Could you please patch `meta/action_groups.yml` instead. This way we don't need to pass the `hostname`, `username`, `validate_certs` and `password` keys again and again.